File: desktopclient/src/main/scala/com/goexp/galgame/gui/task/game/panel/group/node/BrandItem.java
package com.goexp.galgame.gui.task.game.panel.group.node;

import com.goexp.galgame.gui.model.Brand;

public class BrandItem extends DefaultItem {

    public final Brand brand;

    public BrandItem(String title, int count, Brand brand) {
        super(title, count);
        this.brand = brand;
    }

}

File: desktopclient/src/main/scala/com/goexp/galgame/gui/task/game/panel/group/node/BrandItem.scala
package com.goexp.galgame.gui.task.game.panel.group.node

import com.goexp.galgame.gui.model.Brand

class BrandItem(private[this] val title: String,
                private[this] val count: Int,
                val brand: Brand) extends DefaultItem(title, count) {
}